fusesoc/coremanager.py

```diff
@@ -144,8 +150,7 @@ def eq_vln(this, that):
             repo.add_package(package)
 
         request = Request()
-        simplevlnvs = top_core.simpleVLNVs()
-        for sv in simplevlnvs:
+        for _ in top_core.simpleVLNVs():
```

Comment:
What on earth is this loop supposed to do? I have no memory of what my intentions were but it doesn't make sense now

Reply:
@imphil Did we ever figure out what the hell this is?

Reply:
Unclear indeed. Let me have a closer look in the coming days (forgot about it totally).

Reply:
Looks like useless code indeed. I removed it.